python/lsst/eotest/sensor/BFTask.py

```diff
@@ -0,0 +1,271 @@
+        output.append(fits.PrimaryHDU())
+        colnames = []
+        units = []
+        columns = []
+        for amp in all_amps:
+            colnames.extend(['AMP%02i_xcorr' % amp, 'AMP%02i_xcorr_err' % amp, 'AMP%02i_ycorr' % amp,
+            'AMP%02i_ycorr_err' % amp,'AMP%02i_MEAN' % amp])
+            units.extend(['Unitless', 'Unitless','Unitless','Unitless','ADU'])
+            columns.extend([np.array(BFResults[amp][0], dtype=np.float),
+                            np.array(BFResults[amp][1], dtype=np.float),
+                            np.array(BFResults[amp][2], dtype=np.float),
+                            np.array(BFResults[amp][3], dtype=np.float),
+                            np.array(BFResults[amp][4], dtype=np.float)])
+        formats = 'E'*len(colnames)
+        fits_cols = [fits.Column(name=colnames[i], format=formats[i],
+                                 unit=units[i], array=columns[i])
+                     for i in range(len(columns))]
+        output.append(fitsTableFactory(fits_cols))
+        output[-1].name = 'BF_STATS'
+        output[0].header['NAMPS'] = len(all_amps)
+        fitsWriteto(output, outfile, clobber=True)
+        #Output a file of the coefficients at a given mean, given
+        #as the index of the exposure in the list.
+
+        results_file = os.path.join(self.config.output_dir,
+                                        '%s_eotest_results.fits' % sensor_id)
+
+
+        results = EOTestResults(results_file, namps=len(all_amps))
+
+        for amp in all_amps:
+            results.add_seg_result(amp, 'xcorr', BFResults[amp][0][meanidx])
+            results.add_seg_result(amp, 'xcorr_err', BFResults[amp][1][meanidx])
+            results.add_seg_result(amp, 'ycorr', BFResults[amp][2][meanidx])
+            results.add_seg_result(amp, 'ycorr_err', BFResults[amp][3][meanidx])
+            results.add_seg_result(amp, 'MEAN', BFResults[amp][4][meanidx])
+
+
+        results.write(clobber=True)
+
+    def prep_image(self,exp,dark_image):
+        """
+        Crop the image to avoid edge effects based on the Config border
+        parameter. Additionally, if there is a dark image, subtract.
+        """
+
+        # Clone so that we don't modify the input image.
+        local_exp = exp.clone()
+        del exp
+
+        # The border that we wish to crop.
+        border = self.config.nPixBorder
+
+        sctrl = afwMath.StatisticsControl()
+
+        # If a dark image is passed, subtract it.
+        if dark_image:
+            local_exp -= dark_image.getImage()
+
+        # Calculate the mean of the image.
+        mean = afwMath.makeStatistics(local_exp[border:-border, border:-border],
+                                                            afwMath.MEAN, sctrl).getValue()
+        # Crop the image.
+        local_exp = local_exp[border:-border, border:-border]
+
+        return local_exp, mean
+
+def crossCorrelate(maskedimage1, maskedimage2, maxLag, border,sigma,binsize):
```

Comment:
`border` doesn't appear to be used anywhere in this function.  If it's not needed, please remove it as argument.